nodebuilder/share/share.go

```diff
@@ -112,12 +126,30 @@ type module struct {
 	hs     headerServ.Module
 }
 
+// TODO(@Wondertan): break
```

Comment:
I guess we can remove the TODO for now and park the break into the issue.

Reply:
Done https://github.com/celestiaorg/celestia-node/issues/3951